airflow/utils/dag_processing.py

```diff
@@ -43,16 +46,18 @@
 
 # To avoid circular imports
 import airflow.models
-from airflow import configuration as conf
+from airflow import configuration as conf, configuration
 from airflow.dag.base_dag import BaseDag, BaseDagBag
-from airflow.exceptions import AirflowException
+from airflow.exceptions import AirflowException, AirflowDagCycleException
 from airflow.models import errors
 from airflow.settings import logging_class_path
 from airflow.utils import timezone
 from airflow.utils.db import provide_session
 from airflow.utils.log.logging_mixin import LoggingMixin
 from airflow.utils.state import State
+from airflow.utils.timeout import timeout
 
+_log = logging.root.getChild(__package__)
```

Comment:
I think, you should use `LoggingMixin'.

Reply:
Can I also use this when it's not a class? Or should I make a class with just a static method then?

Reply:
Look at: 
https://github.com/apache/airflow/blob/53b89b98371c7bb993b242c341d3941e9ce09f9a/airflow/api/__init__.py#L30

Reply:
thank you